eliot-service/tests/test_cache_manager.py

```diff
@@ -403,3 +410,166 @@ def test_nested_directories(cm_client, tmpdir):
 
     cm.run_once()
     assert cm.lru == {str(file2): 4, str(file3): 2}
+
+
+# NOTE(willkg): If this changes, we should update it and look for new things that should
+# be scrubbed. Use ANY for things that change between tests.
+BROKEN_EVENT = {
```

Comment:
I wonder if in the future we can write something to make testing more straightforward, like being able to declare a template for this to be used in the different tests across the app.